### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Neviothr**
**When merged this pull request will:**
- Do some cleanup.

### CODE REVIEW

**Comment by commy2** [comment]
\(O_O)/

**Comment by commy2** [comment]
>All checks have failed 
Oh well.

File: addons/common/fnc_getUnitAnim.sqf [hunk 1]
```diff
@@ -46,19 +46,19 @@ if (vehicle _unit!= _unit) then {
...
+            case "perc": {"stand"};
+            default {"kneel"};
         };
     };
     _umov = switch (_movstr) do {
-        case "mstp": { "stop" };
-        case "mwlk": { "slow" };
-        case "mrun": { "normal" };
-        case "meva": { "fast" };
-        default { "stop" };
```
Comment (jonpas) [inline addons/common/fnc_getUnitAnim.sqf]: Removed brace.

File: addons/help/gui.hpp [hunk 2]
```diff
@@ -42,13 +40,13 @@ class RscDisplayMain: RscStandardDisplay {
...
-            colorBackgroundActive[] = {0,0,0,0};
-            colorFocused[] = {0,0,0,0};
-            soundClick[] = {"",1,1};
+            colorText[] = {1, 1, 1, 0};
+            colorDisabled[] = {1, 1, 1, 0};
+            colorBackground[] = {0, 0, 0, 0};
+            colorBackgroundDisabled[] = {0, 0, 0, 0};
+            colorBackgroundActive[] = {0, 0, 0, 0};
+            colorFocused[] = {0, 0, 0, 0};
+            soundClick[] = {"", 1, 1};
```
Comment (commy2) [inline addons/help/gui.hpp]: I like this more without spaces in config. That's what all the debin tools do and how they show up in the config viewer.

File: addons/jr/cfgweapons.hpp [hunk 3]
```diff
@@ -66,7 +62,6 @@ class CfgWeapons {
     };
 
     class LRR_base_F: Rifle_Long_Base_F {};
-
     class srifle_LRR_F : LRR_base_F {
```
Comment (commy2) [inline addons/jr/cfgweapons.hpp]: Might as well also remove the white space before the :

File: addons/common/fnc_addMagazine.sqf [hunk 4]
```diff
@@ -51,7 +51,7 @@ if (!isClass _config || {getNumber (_config >> "scope") < 2}) exitWith {
 if (_verify) then {
     if (_unit canAdd _item) then {
         if (_ammo < 0) then {
-            _unit addMagazine [_item, 1E6]; // addMagazine STRING is not synched when used on remote units. addMagazine ARRAY is.
+            _unit addMagazine [_item, 1E6]; // addMagazine STRING is not synched when used on remote units. addMagazine ARRAY is. Wtf BI?
```
Comment (Killswitch00) [inline addons/common/fnc_addMagazine.sqf]: Remove

File: addons/common/fnc_modelHeadDir.sqf [hunk 5]
```diff
@@ -42,8 +42,8 @@ Author:
 
 #include "script_component.hpp"
 SCRIPT(modelHeadDir);
-private["_pilotPos", "_neckPos", "_polar", "_pitch", "_abs", "_dif", "_world", "_return"];
 
+private["_pilotPos", "_neckPos", "_polar", "_pitch", "_abs", "_dif", "_world", "_return"];
```
Comment (Killswitch00) [inline addons/common/fnc_modelHeadDir.sqf]: `private ["_pilotPos", ...`

File: addons/common/fnc_randPosArea.sqf [hunk 6]
```diff
@@ -62,8 +62,8 @@ if (_isRect) then {
     private _phi = random 360;
 
     // Scale circle to dimensions of the ellipse
-    private _x = sqrt(_rho) * cos(_phi);
-    private _y = sqrt(_rho) * sin(_phi);
+    private _x = sqrt(_rho) * cos (_phi);
```
Comment (Killswitch00) [inline addons/common/fnc_randPosArea.sqf]: Inconsistency in style between `sqrt` and cos` or `sin` (below)

File: addons/common/fnc_switchPlayer.sqf [hunk 7]
```diff
@@ -40,18 +40,20 @@ _oldUnit = player;
 _ar = [weapons _oldUnit, magazines _oldUnit, rank _oldUnit, score _oldUnit, group _oldUnit, getPos _oldUnit, (leader _oldUnit) == _oldUnit, vehicleVarName _oldUnit];
 
 _dummyGroup = createGroup (side _oldUnit);
-if (isNull _dummyGroup) exitWith { hint "Sorry, something went wrong, dummyGroup is null" };
+
+if (isNull _dummyGroup) exitWith {hint "Sorry, something went wrong, dummyGroup is null"};
```
Comment (Killswitch00) [inline addons/common/fnc_switchPlayer.sqf]: Add empty line after to keep the style introduced in the changes below

File: addons/strings/fnc_trim.sqf [hunk 8]
```diff
@@ -14,8 +14,8 @@ Returns:
 
 Example:
     (begin example)
-        _result = [" frogs are fishy   "] call CBA_fnc_trim;
-        // _result => "frogs are fishy"
+        _result = [" commy smells of fish   "] call CBA_fnc_trim;
```
Comment (Killswitch00) [inline addons/strings/fnc_trim.sqf]: Restore the original

File: addons/ui/flexiMenu/data/menu_buttonList.hpp [hunk 9]
```diff
@@ -1,47 +1,47 @@
...
+ //#include "common_listClass.hpp"
 #define _imagePath2(TOKEN) QUOTE(PATHTOF(flexiMenu)\data\buttonList\TOKEN.paa)
 
     class listButton: _flexiMenu_RscShortcutButton {
         x = 0.5;
         y = 0.5;
-        w = 0; //_SMW; // hide initially
-        //w = _SMW;
+        w = 0;  //_SMW;  // hide initially
+         //w = _SMW;
```
Comment (Killswitch00) [inline addons/ui/flexiMenu/data/menu_buttonList.hpp]: restore original line

File: addons/ui/flexiMenu/data/menu_buttonList.hpp [hunk 10]
```diff
@@ -136,21 +135,20 @@ class CBA_flexiMenu_rscButtonList { //: _flexiMenu_rscRose
         ExpandMacro_RowControls(15);
         ExpandMacro_RowControls(16);
         ExpandMacro_RowControls(17);
-        //-----------------------
+         //-----------------------
```
Comment (Killswitch00) [inline addons/ui/flexiMenu/data/menu_buttonList.hpp]: Don't indent comments

File: addons/ui/flexiMenu/data/menu_iconRow.hpp [hunk 11]
```diff
@@ -1,50 +1,50 @@
...
+class CBA_flexiMenu_rscIconRow {  //: _flexiMenu_rscRose
+    idd = -1;  //_flexiMenu_IDD;
     movingEnable = 0;
     onLoad = QUOTE(with uiNamespace do {GVAR(display) = _this select 0};);
     onUnload = QUOTE(with uiNamespace do {GVAR(display) = displayNull};);
     class controlsBackground {};
     class objects {};
 
-    // custom flexiMenu properties
+     // custom flexiMenu properties
```
Comment (Killswitch00) [inline addons/ui/flexiMenu/data/menu_iconRow.hpp]: restore original

File: addons/ui/flexiMenu/data/menu_iconRow.hpp [hunk 12]
```diff
@@ -97,31 +97,29 @@ class CBA_flexiMenu_rscIconRow { //: _flexiMenu_rscRose
...
         animTextureFocused = _imagePath(mid_button_focus);
         animTexturePressed = _imagePath(mid_button_down);
-        animTextureDefault = _imagePath(mid_button_normal); // used?
-        animTextureNoShortcut = _imagePath(mid_button_normal); // used?
+        animTextureDefault = _imagePath(mid_button_normal);  // used?
+        animTextureNoShortcut = _imagePath(mid_button_normal);  // used?
         //action = _eval_action(-1);
     };
-    //---------------------------------
+     //---------------------------------
```
Comment (Killswitch00) [inline addons/ui/flexiMenu/data/menu_iconRow.hpp]: lots of these...

File: addons/ui/flexiMenu/data/menu_popup.hpp [hunk 13]
```diff
@@ -51,34 +51,33 @@ class CBA_flexiMenu_rscPopup { //: _flexiMenu_rscRose
...
+        animTextureNormal = _imagePath(normal);
+        animTextureDisabled = _imagePath(disabled);
+        animTextureOver = _imagePath(over);
+        animTextureFocused = _imagePath(focused);
+        animTexturePressed = _imagePath(down);
+        animTextureDefault = _imagePath(default);
+        animTextureNoShortcut = _imagePath(normal);
     };
-    //---------------------------------
+     // ---------------------------------
```
Comment (Killswitch00) [inline addons/ui/flexiMenu/data/menu_popup.hpp]: ...

Comment (Killswitch00) [inline addons/ui/flexiMenu/data/menu_popup.hpp]: Dont indent the start of the comment //

File: addons/ui/flexiMenu/data/menu_popup.hpp [hunk 14]
```diff
@@ -100,22 +99,21 @@ class CBA_flexiMenu_rscPopup { //: _flexiMenu_rscRose
         ExpandMacro_RowControls(17);
         ExpandMacro_RowControls(18);
         ExpandMacro_RowControls(19);
-        //-----------------------
+         // -----------------------
```
Comment (Killswitch00) [inline addons/ui/flexiMenu/data/menu_popup.hpp]: same

File: addons/ui/flexiMenu/data/menu_rose.hpp [hunk 15]
```diff
@@ -38,7 +38,7 @@ class CBA_flexiMenu_rscRose {
     class controlsBackground {};
     class objects {};
 
-    // custom flexiMenu properties
+     //custom flexiMenu properties
```
Comment (Killswitch00) [inline addons/ui/flexiMenu/data/menu_rose.hpp]: Restore original line

File: addons/ui/flexiMenu/fnc_highlightCaretKey.sqf [hunk 16]
```diff
@@ -9,16 +9,10 @@ in order to avoid the millions of Deprecated function warnings:
...
 
-private ['_array', '_index', '_offset', '_hotKeyColor_ST',
-    '_captionArray', '_i', '_len', '_array2'];
-
-_array = _this select 0;
-_index = _this select 1;
-_offset = _this select 2;
-_hotKeyColor_ST = _this select 3; // (ST=structured text) "<t color='#f07EB27E'>"
+private ["_array", "_index", "_offset", "_hotKeyColor_ST", "_captionArray", "_i", "_len", "_array2"];
+params ["_array", "_index", "_offset", "_hotKeyColor_ST"]; // (ST=structured text) "<t color='#f07EB27E'>"
```
Comment (Killswitch00) [inline addons/ui/flexiMenu/fnc_highlightCaretKey.sqf]: Redundant private declarations

File: addons/ui/flexiMenu/fnc_keyUp.sqf [hunk 17]
```diff
@@ -1,13 +1,8 @@
...
 #include "\x\cba\addons\ui_helper\script_dikCodes.hpp"
 
-private["_handled", /* "_ctrl", */ "_dikCode", "_shift", "_ctrlKey", "_alt",
-    "_active", "_potentialKeyMatch"];
-
-_dikCode = _this select 1;
-_shift = _this select 2;
-_ctrlKey = _this select 3;
-_alt = _this select 4;
+private ["_handled", /* "_ctrl", */ "_dikCode", "_shift", "_ctrlKey", "_alt", "_active", "_potentialKeyMatch"];
```
Comment (Killswitch00) [inline addons/ui/flexiMenu/fnc_keyUp.sqf]: Some redundant private declarations

File: addons/versioning/XEH_postInitServer.sqf [hunk 18]
```diff
@@ -18,23 +18,53 @@ FUNC(paranoid) = {
...
-    [format["%1 - Not running! (Machine: %2)", _mod, _machine], QUOTE(COMPONENT), [CBA_display_ingame_warnings, true, true]] call CBA_fnc_debug;
+    [format ["%1 - Not running! (Machine: %2)", _mod, _machine], QUOTE(COMPONENT), [CBA_display_ingame_warnings, true, true]] call CBA_fnc_debug;
 };
 
 QGVAR(mismatch) addPublicVariableEventHandler { (_this select 1) call FUNC(handleMismatch) };
 
 private "_str";
-_str = 'if(isServer)exitWith{};if (isNil "CBA_display_ingame_warnings") then { CBA_display_ingame_warnings = true; };LOG("cba_versioning_check");0 = objNull spawn { sleep 1; sleep 1; _func={GVAR(mismatch)=[format["%2 (%1)",name player, player],_this];publicVariable QGVAR(mismatch);_this spawn{_t=format["You are missing the following mod: %1",_this];diag_log text _t;sleep 2;if (CBA_display_ingame_warnings) then {player globalChat _t}}};';
+
+_str = 'if (isServer) exitWith{};\
```
Comment (Killswitch00) [inline addons/versioning/XEH_postInitServer.sqf]: This string was compacted for network performance reasons. Lack of proper formatting is intentional (and could be improved by removing even more spaces, actually)

**Comment by commy2** [comment]
Can we split this up into one PR for each component?

**Comment by Neviothr** [comment]
@commy2 - will do.

**Pull request closed without merging**